versions/3.0.md

```diff
@@ -1820,7 +1820,7 @@ The key value used to identify the callback object is an expression, evaluated a
 ##### Patterned Fields
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/examples/v3.0/callback-example.yaml) is available.
+<a name="callbackExpression"></a>{expression} | [Path Item Object](#pathItemObject) | A Path Item Object used to define a callback request and expected responses.  A [complete example](/examples/v3.0/callback-example.yaml) is available.
```

Comment:
The link in my fork points to https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/examples/v3.0/callback-example.yaml - not the master branch? Viewing the changes I see a link to the [commit blob](https://github.com/MikeRalphson/OpenAPI-Specification/blob/7dd9891b20ceb7232c9704dbc7b3f7bbff30204e/examples/v3.0/callback-example.yaml)